CHANGELOG.md

```diff
@@ -2,86 +2,63 @@
 
 ## Release (1.3.0)
 
-### Ajouts généraux 
-### Commandes ajoutées
+In this release, we decided to use English as the main language.
 
-### Corrections de bogues
+### additions
 
-- Correction du `SkipTitle` en cas de "test de combat" (@Jauke)
+### Bug fixes
+- Fix `SkipTitle` in the "battle-test-case" (@Jauke)
 
 ## Release (1.2.0)
 
-### Ajouts généraux
-
-* Gestion des Intervalles dans les variables locales : `SV[1, 2, 1..18] = 10` (par exemple) par @xvw
-
-### Commandes ajoutées
-
-* `camera_scrolling?` (@xvw, reporté par @Zer0Zer0x)
-* `windowskin_tone` et `windowskin_opacity` (@xvw, reporté par @YendaHusk)
-* `event_tone` et `player_tone` (@xvw, reporté par @Zer0Zer0x)
-* `event_character_name`, (@BastienDuplessier)
-* `event_character_index`, (@BastienDuplessier)
-* `event_change_character`, (@BastienDuplessier)
-*    - `menu_enabled?|disabled?` 
```

Comment:
The leading bullet's symbol (`*`) should be removed. Could you remove the **trailing whitespace** as well ?